src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -165,11 +184,16 @@ private static bool HasBuiltInImplicitConversion(ITypeSymbol argumentType, IType
             var typeName = targetTypeSymbol.GetFullMetadataName();
             var targetType = IntrinsicTypeConverters.FirstOrDefault(t => t.type.FullName == typeName);
 
-            if (targetType.typeConverter == null)
+            if (targetType == default((Type type, Lazy<TypeConverter> typeConverter)))
```

Comment:
```suggestion
            if (targetType == default)
```


(if it was added for clarity, feel free to ignore)